helm/prometheus-rules/templates/_helpers.tpl

```diff
@@ -44,31 +44,31 @@ phoenix
 {{- end -}}
 
 {{- define "isCertExporterInstalled" -}}
-{{- if has .Values.managementCluster.provider.kind (list "openstack" "vsphere") -}}
+{{- if has .Values.managementCluster.provider.kind (list "openstack" "vcd" "vsphere") -}}
 false
 {{- else -}}
 true
 {{- end -}}
 {{- end -}}
 
 {{- define "isClusterServiceInstalled" -}}
-{{- if has .Values.managementCluster.provider.kind (list "openstack" "vsphere") -}}
+{{- if has .Values.managementCluster.provider.kind (list "openstack" "vcd" "vsphere") -}}
 false
 {{- else -}}
 true
 {{- end -}}
 {{- end -}}
 
 {{- define "isVaultBeingMonitored" -}}
-{{- if has .Values.managementCluster.provider.kind (list "openstack" "vsphere") -}}
+{{- if has .Values.managementCluster.provider.kind (list "openstack" "vcd" "vsphere") -}}
 false
 {{- else -}}
 true
 {{- end -}}
 {{- end -}}
 
 {{- define "isBastionBeingMonitored" -}}
-{{- if has .Values.managementCluster.provider.kind (list "openstack" "vsphere") -}}
+{{- if has .Values.managementCluster.provider.kind (list "openstack" "vcd" "vsphere") -}}
```

Comment:
Possibly this should be based on negation at this point

Reply:
Should we maybe create a isOnPremise property up the config chain instead ?  Or maybe something like location: cloud || on premise and that property is calculated from the provider ?